src/firefly/js/ui/PopupUtil.jsx

```diff
@@ -119,22 +113,19 @@ export function hideInfoPopup() {
     dispatchHideDialog(INFO_POPUP);
 }
 
-function makeContent(content, isCopyable, wrapperStyle={}) {
+function makeContent(content, sx={}) {
     return (
-        <div style={{padding:5}}>
-            <div style={{minWidth:350, maxWidth: 500, padding:'10px 10px 15px 10px', fontSize:'120%', overflow: 'hidden', ...wrapperStyle}}
-                 className={isCopyable ? 'popup-panel-textcopyable' : ''}>
-                {content}
-            </div>
-            <div style={{padding:'0 0 5px 10px'}}>
-                <CompleteButton dialogId={INFO_POPUP} />
-            </div>
-        </div>
+        <Stack sx={{px:2, py:1}} spacing={2}>
+            <Stack sx={{minWidth:350, maxWidth: 500, overflow: 'hidden', ...sx}} >
```

Comment:
`sx` should not be passed in.  User would have to look at the code to see where it would applied.
Instead, use className to define elements which can be modified.
```
        <Stack px={2}  py={1} spacing={2}>
            <Stack className='FF-Popup-Content' minWidth={350} maxWidth={500} sx={{overflow: 'hidden'}} >
```
This way, a user of the component can target the content's element and apply any style he/she wishes.
Notice I also move `px` and `py` out of the `sx` prop?  They are what Joy UI called System props.  They can be used directly in all of their utility components.  It may look cleaner.

Reply:
Thanks, I was only mildly aware of this. I will go back over the code and use them more directly. I was using them some with `Stack` but I encountered a case where it did not work as expect and starting giving me (a lot) of react warnings. I will start changing the code and see it I can isolate that case.

Reply:
>  <Stack px={2}  py={1} spacing={2}>
            <Stack className='FF-Popup-Content' minWidth={350} maxWidth={500} sx={{overflow: 'hidden'}} >

I don't think I was understand what you were saying from previous conversations. I thought you always wanted `sx` passed. However, this does make sense to use `className` more aggressively. I think I am still trying to get my head around the idea that you can access styles using `sx`. That is far more powerful than what we have done previously just passing in styles. I do like the idea of being able to access styles in the code.